CS483/CS483/Kartaclysm/Components/ComponentKartController.cpp

```diff
@@ -462,13 +496,17 @@ namespace Kartaclysm
 
 	void ComponentKartController::ArmorPlate(int p_iArmorStack)
 	{
-		// TODO: I think maybe there should only actually be 4 stacks *shrug*
 		m_iMaxSpeedCoreStat = 1 + p_iArmorStack;
 		m_iAccelerationCoreStat = 5 - p_iArmorStack;
 		m_iHandlingCoreStat = 5 - p_iArmorStack;
```

Comment:
The 5 here is to do with the stats themselves, not the ability charges. This part is as it should be.